docs/conf.py

```diff
@@ -52,8 +53,9 @@
 
 # General information about the project.
 project = 'Django REST Framework JSON API'
-copyright = '2015, Jerel Unruh and contributors'
-author = 'Jerel Unruh'
+year = datetime.date.today().year
+copyright = '{}, Django REST Framework JSON API contributors'.format(year)
```

Comment:
Also, I would think @jerel's name should be retained with special emphasis as the original creator. "and contributors" for the rest...

Reply:
Oh wait, git says @gaker for the initial commit. Hmm... I'm still uncomfortable with removing a specific name from a copyright statement.